libs/go/sia/gcp/functions/gcf_get_sia_cert.go

```diff
@@ -0,0 +1,263 @@
+package functions
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"github.com/AthenZ/athenz/clients/go/zts"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+// GcfGetSiaCerts this method can be called from within a GCF (Google Cloud Function) -
+//
+//	to get an Athenz certificate from ZTS.
+//
+// This file should usually be copied without changes into the GCF source-code.
```

Comment:
not sure what this is referring to. nobody should copy the source file. instead, they'll just import the module and call the methods, so let's remove this comment

Reply:
Archeological remnant removed...